### PR TITLE
 enabling kubernetes access and APIM CRDS via cil tool

### PR DESCRIPTION
**Pull request by Shehanir**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: import-export-cli/box/resources/kubernetes_resources/api_cr.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: wso2.com/v1alpha1
+kind: API
+metadata:
+  name:
+spec:
+  definition:
+    configmapName:
+    type: swagger
+  replicas:
+  mode: privateJet
```
Comment (menakaj) [inline import-export-cli/box/resources/kubernetes_resources/api_cr.yaml]: Add a newline at the end of the file.

Comment (Shehanir) [reply import-export-cli/box/resources/kubernetes_resources/api_cr.yaml]: sure

File: import-export-cli/cmd/addApi.go [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+var interceptorsConfName string
+
+// addApiCmd represents the addApi command
+var addCmd = &cobra.Command{
+	Use:     addCmdLiteral,
+	Short:   addCmdShortDesc,
+	Long:    addCmdLongDesc,
+	Example: addCmdExamples,
+}
+var addApiCmd = &cobra.Command{
```
Comment (dinusha92) [inline import-export-cli/cmd/addApi.go]: Keep a new line and add a method comment. :)

Comment (Shehanir) [reply import-export-cli/cmd/addApi.go]: sure

File: import-export-cli/cmd/addApi.go [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+const apiLongDesc = `Add, Update and Delete APIs in kubernetes cluster. JSON and YAML formats are accepted.
+available modes are as follows
+* kubernetes`
+const apiExamples = utils.ProjectName + " add/update " + apiCmdLiteral + ` -n petstore --from-file=./Swagger.json --replicas=3 --namespace=wso2`
+const addCmdLiteral = "add"
+const addCmdShortDesc = "Add an API to the kubernetes cluster"
+const addCmdLongDesc = `Add an API from a Swagger file to the kubernetes cluster. JSON and YAML formats are accepted.
+available modes are as follows
+* kubernetes`
+const addCmdExamples = utils.ProjectName + " " + apiCmdLiteral + " " + `-n petstore --from-file=./Swagger.json --replicas=3 --namespace=wso2`
```
Comment (dinusha92) [inline import-export-cli/cmd/addApi.go]: This should be apimcli add api (now it shows as apimcli api ). Shall we correct it?

Comment (Shehanir) [reply import-export-cli/cmd/addApi.go]: sure

**Pull request merged**